morphapi/paths_manager.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, base_dir=None, **kwargs):
         # Get and make base directory
 
         if base_dir is None:
-            self.base_dir = Path.home() / ".morphapi"
+            self.base_dir = Path.home() / ".brainglobe" / "morphapi"
         else:
             self.base_dir = Path(base_dir)
 
```

Comment:
Think we need a `parents=True` argument to `mkdir` now, otherwise looks good to me.

Reply:
Done, good catch!